src/main/java/fr/alasdiablo/janoeo/config/FrequencyConfig.java

```diff
@@ -21,299 +24,316 @@
     }
 
     public static class Config {
-
-        public final IntValue COPPER_ORE_SIZE, COPPER_ORE_COUNT, COPPER_ORE_BOTTOM, COPPER_ORE_TOP;
-
-        public final IntValue TIN_ORE_SIZE, TIN_ORE_COUNT, TIN_ORE_BOTTOM, TIN_ORE_TOP;
+        // --------------------------------------- overworld.stone --------------------------------------
+        // --> MODDED ORE
+        public final HashMap<String, HashMap<String, HashMap<String, IntValue>>> Frequency = new HashMap<String, HashMap<String, HashMap<String, IntValue>>>();
```

Comment:
+ Do not use HashMap as base variable type (use Map instead)
+ The key of this Map, make the mod less modular
+ `new HashMap<String, HashMap<String, HashMap<String, IntValue>>>()` can be replaced by `new HashMap<>()` we already know the type
+ Variable name like a class ?